portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,45 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  /* margin-left: auto;
+  margin-right: auto; */
+  width: 100%;
 }
 
-#greeting-container {
+/* #greeting-container {
   margin-top: 20px;
+} */
+
+body {
+  background-image: url('images/background4.jpeg');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
```

Comment:
Order the css properties in alphabetical order. It is easier to look then up if they are organized. Do the same for other css rules in this file

Reply:
done